docker-compose.yml

```diff
@@ -4,36 +4,78 @@ services:
     image: postgres:9.3
     container_name: api-java-postgres
 
-  possum:
-    image: registry.tld/possum:0.1.0-stable
-    container_name: api-java-possum
+  conjur:
+    image: cyberark/conjur:0.1.0-stable
+    container_name: api-java-conjur
     command: server -a cucumber
     environment:
+      PORT: 3000
```

Comment:
Do we have to set the port here? Seems redundant with `ports` and `expose` below.

`expose` should really be `EXPOSE` in the Dockerfile, so I'd rather not use it here, confuses the issue.

Reply:
The first port is environment variable for the conjur server to start listening on port 3000 instead of the default 80
I can change the scripts to access by 80 and change the ports mapping of conjur server to 3000:80, if you will decide its necessary 

Waiting to your reply...